From 134d9bff83e4248781b7dac72f1706dce76df086 Mon Sep 17 00:00:00 2001
From: Piotr Gorski <lucjan.lucjanov@gmail.com>
Date: Mon, 13 Jun 2022 18:25:06 +0200
Subject: [PATCH 31/32] Revert "lrng-5.18: drop
 EXPORT_SYMBOL_GPL(add_bootloader_randomness)"

This reverts commit 00d3722e57a8e11140375153e9c64178401ab481.

Signed-off-by: Piotr Gorski <lucjan.lucjanov@gmail.com>
---
 drivers/char/lrng/lrng_interface_random_kernel.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/char/lrng/lrng_interface_random_kernel.c b/drivers/char/lrng/lrng_interface_random_kernel.c
index ca2efefb1..81060ba98 100644
--- a/drivers/char/lrng/lrng_interface_random_kernel.c
+++ b/drivers/char/lrng/lrng_interface_random_kernel.c
@@ -108,6 +108,7 @@ void add_bootloader_randomness(const void *buf, size_t size)
 			     IS_ENABLED(CONFIG_RANDOM_TRUST_BOOTLOADER) ?
 			     size * 8 : 0);
 }
+EXPORT_SYMBOL_GPL(add_bootloader_randomness);
 
 /*
  * Callback for HID layer -- use the HID event values to stir the entropy pool
-- 
2.36.1.203.g1bcf4f6271

